scripts/ci/test-operator

```diff
@@ -55,9 +55,10 @@ mkdir -p "$CR_DIR"
 # Organize expected dir structure for the registry image build.
 operator-courier nest "${OP_PATH}" "${DEPLOY_DIR}/${PKG_NAME}"
 pushd "$DEPLOY_DIR"
-NAMESPACE="operators"
+NAMESPACE="$PKG_NAME"
```

Comment:
The namespace here should also depend on the operator's InstallModes:
If AllNamespaces is supported, leave as "operators"
elif SingleNamespaces is supported, set as "$PKG_NAME" and create an OperatorGroup for that namespace
else, error out early